rd_ui/app/views/query.html

```diff
@@ -91,6 +91,9 @@ <h4 class="modal-title">Query Archive</h4>
     <div class="col-lg-12 p-5 bg-orange c-white" ng-if="query.is_archived">
       This query is archived and can't be used in dashboards, and won't appear in search results.
     </div>
+    <div class="col-lg-12 p-5 bg-orange c-white" ng-if="query.is_draft">
+      This query is a draft.
```

Comment:
You can [allow edit from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) and I will add it.